src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import 'utils/variables';
```

Comment:
Ensure that the file `utils/variables.scss` exists and contains the necessary variable definitions, such as `$main-text-color` and `$accent-color`, which are used in `card.scss`.